src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import styles from './CourseInfo.module.css';
+
+import Button from '../../common/Button/Button';
+import { mockCourses } from '../../data/data';
+import getCoursesDuration from '../../helpers/getCoursesDuration';
+import getAuthorNames from '../../helpers/getAuthorNames';
+
+const CourseInfo = () => {
+  let { courseId } = useParams();
+  let title;
+  let description;
+  let duration;
+  let authors;
+  let creationDate;
+  mockCourses.forEach((element) => {
+    if (courseId == element.id) {
+      title = element.title;
+      description = element.description;
+      duration = element.duration;
+      authors = element.authors;
+      creationDate = element.creationDate;
+    }
+  });
```

Comment:
Well, this is not correct idea: right now per each render you are going to look through the mocks and find appropriate item. This is not really good since performance point and also scalability of your component. I would like you to use `useState` for storing found course object and put this item in scope of the `useEffect`. In this way you don't have to use lots of variables.

Reply:
This problem will be solved in next Task.